src/main/java/net/mcreator/preferences/data/BlocklySection.java

```diff
@@ -43,7 +43,7 @@ public class BlocklySection extends PreferencesSection {
 	BlocklySection(String preferencesIdentifier) {
 		super(preferencesIdentifier);
 
-		blockRenderer = addEntry(new StringEntry("blockRenderer", "Thrasos", "Geras", "Thrasos"));
+		blockRenderer = addEntry(new StringEntry("blockRenderer", "Thrasos", "Geras", "Thrasos", "Zelos"));

```

Comment:
While we are at it, I would put Thrasos first since it is the default value. It is a bit strange having it in the middle of the list as default value :D

Reply:
I thought it was alphabetic order 😅